### ISSUE TITLE
requirements list will discourage people from using the rester

### ISSUE THREAD

**Issue by ardunn** [materialsintelligence/matscholar#61]
If we want people to use the awesome API we have made, the rester needs to be easily installed. If you don't have 3.6, its not easy to install.  

I get that the extra requirements are for the non-rester parts of the code. But if they make installing the feature that the most people will be interested in (likely the rester to access the API), then they are not serving much good here. Maybe we could relocate them to another package, or have mat scholar's rester be its own package, `matscholar-rest` or something like that, with a minimal set of requirements and which we can easily put on pipit.

**Comment by jdagdelen** [comment]
Fair enough. I think we should just train our own tokenizer and get rid of CDE though.

On Mon, Nov 25, 2019 at 8:14 PM, Alex Dunn <notifications@github.com> wrote:
If we want people to use the awesome API we have made, the rester needs to be easily installed. If you don't have 3.6, its not easy to install.

I get that the extra requirements are for the non-rester parts of the code. But if they make installing the feature that the most people will be interested in (likely the rester to access the API), then they are not serving much good here. Maybe we could relocate them to another package, or have mat scholar's rester be its own package, matscholar-rest or something like that, with a minimal set of requirements and which we can easily put on pipit.

**Comment by mkhorton** [comment]
+1 for this, I'll eventually be getting around to adding MatScholar integration on MP, and the requirements are definitely heavy (CDE aside, there's the direct git install package, gensim, etc.)

**Comment by ardunn** [comment]
That too.

**Comment by ardunn** [comment]
@jdagdelen we can discuss this on the call today if there is time.
